src/main/java/org/jetbrains/research/refactorinsight/data/RefactoringEntry.java

```diff
@@ -82,8 +85,35 @@ public void setTimeout(boolean timeout) {
    * @param commit       current commit.
    * @return new refactoring entry.
    */
-  public static RefactoringEntry convert(List<Refactoring> refactorings, TimedVcsCommit commit,
-                                         Project project) {
+  public static RefactoringEntry convertJavaRefactorings(List<Refactoring> refactorings, TimedVcsCommit commit,
+                                                         Project project) {
+
+    RefactoringEntry entry =
```

Comment:
We have a bit of duplicated code here and in other places (e.g. in `InfoFactory` and `Handler`), because of dealing with `Refactoring` classes from two different miners. These classes, however, look nearly identical. 
Could it be possible to instead use a common interface for Refactoring, implemented by wrappers for these two classes, and delegate method calls to concrete implementations in these wrappers? Looks like this could save a bit of duplicated code. I may be missing something, though.

Reply:
We should push refactoring entities from each miner through because we need to process each refactoring type in a specific way. See Handler.specify() method implementations.